File: 0125-valid-palindrome/0125-valid-palindrome.java
class Solution {
    public boolean isPalindrome(String s) {
        
        String output = s.replaceAll("[^a-zA-Z0-9]", "").toLowerCase();
        StringBuilder reversed = new StringBuilder(output).reverse();
        if(output.equals(reversed.toString())){
            return true;
        }
            
        return false;
    }
}

File: 0125-valid-palindrome/0125-valid-palindrome.js
/**
 * @param {string} s
 * @return {boolean}
 */
var isPalindrome = function(s) {
    let output = s.toLowerCase();
    let ans = output.replace(/[^a-z0-9]/g,'');
    let start = 0, end = ans.length-1;
    while(start<=end){
        if(ans.charAt(start) !== ans.charAt(end)){
            return false;
        }
        start++;
        end--;
    }
    return true;
};